### DISCUSSION TITLE
[off]Jovem Aprendiz

### DISCUSSION THREAD

**Discussion by ghost** [frontendbr/forum#885] [Discussão]
Me desculpem pelo  pelo off-topic mas é meio que uma questão de urgência, alguém conhece/trabalha em alguma empresa ligada a front-end que esteja abrindo vaga para jovem aprendiz ?

**Comment by renanlara** [comment]
Eu nunca vi nenhum dev trabalhando como jovem aprendiz.
O conselho que dou é o seguinte: procure por agências ou produtoras pequenas. Pequenas mesmo! Que as vezes trabalha um designer, você e o chefe. Lá você vai adquirir experiência, uma vez que adquiriu isso, as portas estarão abertas.

**Comment by giioohbernini** [comment]
Comecei em uma agencia minúscula como o Renan descreveu acima, mas você
deveria procurar por estágios invés de jovem aprendiz, creio eu..

Em sex, 12 de jan de 2018 09:11, Renan Lara <notifications@github.com>
escreveu:

**Comment by felipefialho** [comment]
@pauloLu Tenta ver o https://github.com/frontendbr/vagas

**Comment by AmandaFonseca** [comment]
Eu já vi 1 vez  no Infojobs uma vaga de front-end para  jovem/aprendiz porém, conforme o @renanlara já falou eu também nunca vi nenhum dev trabalhando como jovem aprendiz de fato.

**Comment by synga** [comment]
Cara acho que ovem Aprendiz não seria o termo certo a se utilizar, se você procurar esse termo por ai nas vagas provavelmente não vai encontrar nada a não ser em programas do governo como o CIEE e ainda assim acho difícil arranjar algo especifico, normalmente são trampos de aux. administrativo.
Tente procurar por Trainee, vai no Vagas, Infojobs, Linkedin ou qualquer outro lugar e procure Trainee, as vezes até Junior os caras colocam na descrição querem alguém pra aprender o trampo.
Se estiver precisando por causa de faculdade pra fazer estágio recomendo também falar com coordenador de curso ou ver na secretaria se existe alguém que foi abrir uma vaga na faculdade, muitas delas tem programas onde as pessoas podem anunciar para pegar estagiário.
No demais boa sorte.
